ui/src/components/layout/ContextNews.vue

```diff
@@ -16,10 +16,11 @@
                 <h5>
                     {{ feed.title }}
                 </h5>
+                <!-- fix 3 consider removign tooltip wrapper in DateAgo comp seperate file -->
```

Comment:
If you're planning to use `AI` for all the work, at least remove the generated comments of this sort, otherwise, next time, the PR will be automatically closed.